### PR TITLE
calibre: 2.66.0 -> 2.68.0

### PR DESCRIPTION
**Pull request by cmfwyp**
###### Things done
- [x] Tested using sandboxing
  ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS,
    or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
  - [x] NixOS
  - [ ] OS X
  - [ ] Linux
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of `calibre`
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/applications/misc/calibre/default.nix [hunk 1]
```diff
@@ -129,14 +129,14 @@ stdenv.mkDerivation rec {
     ];
     categories = "Office";
     extraEntries = ''
-      Actions=ebook-edit ebook-viewer
+      Actions=Edit;Viewer;
```
Comment (peterhoeg) [inline pkgs/applications/misc/calibre/default.nix]: Is there a standard somewhere for how the actions are named?

Comment (peterhoeg) [inline pkgs/applications/misc/calibre/default.nix]: Is there a standard somewhere for how the actions are named

**Comment by peterhoeg** [comment]
I know that it's 2 different things in this PR, but please squash the commits.

File: pkgs/applications/misc/calibre/default.nix (hunk 1 above)
Comment (cmfwyp) [inline pkgs/applications/misc/calibre/default.nix]: Kind of: https://specifications.freedesktop.org/desktop-entry-spec/desktop-entry-spec-latest.html#example.

**Pull request merged**